azure-devops/projects/pagopa-projects/pagopa-nodo-pagamenti.tf

```diff
@@ -103,7 +107,8 @@ module "pagopa-nodo4-nodo-dei-pagamenti_deploy" {
   repository                   = var.pagopa-nodo4-nodo-dei-pagamenti.repository
   github_service_connection_id = azuredevops_serviceendpoint_github.io-azure-devops-github-rw.id
 
-  ci_trigger_use_yaml = true
+  ci_trigger_use_yaml           = true
+  pull_request_trigger_use_yaml = true
```

Comment:
```suggestion
```
`pull_request_trigger_use_yaml` not supported as design for deploy pipelines. If you want manual deploy from azure devops u need to remove also `ci_trigger_use_yaml`